languages/react-native/example/src/App.tsx

```
@@ -0,0 +1,34 @@
+
+const session = new USaintSessionBuilder().anonymous();
+
+const client = await new CourseScheduleApplicationBuilder().build(session);
+
+export default function App() {
+  const category = new LectureCategoryBuilder().major(
+    'IT대학',
+    '글로벌미디어학부',
+    undefined
+  );
+  let result = use(client.findLectures(2025, SemesterType.One, category));
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'result' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)